### PR TITLE
ENH build 2.34 for older gcc

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -14,14 +14,14 @@ set -e
 #  done
 #popd
 
-for file in ./crosstool_ng/packages/binutils/${PKG_VERSION}/*.patch; do
+for file in ./crosstool_ng/packages/binutils/$PKG_VERSION/*.patch; do
```
Comment (beckermr) [inline recipe/build.sh:17]:
```suggestion
for file in ./crosstool_ng/packages/binutils/${PKG_VERSION}/*.patch; do
```

File: recipe/install_binutils_symlinks.sh [hunk 2]
```diff
@@ -9,3 +9,4 @@ for tool in addr2line ar as c++filt dwp elfedit gprof ld ld.bfd ld.gold nm objco
 done
 
 ln -s "$PREFIX/bin/ld.gold" "$PREFIX/bin/gold"
+© 2020 GitHub, Inc.
```
Comment (beckermr) [inline recipe/install_binutils_symlinks.sh:12]:
```suggestion
```

File: recipe/install_binutils.sh [hunk 3]
```diff
@@ -5,20 +5,10 @@ set -e
...
-  ln -s $PREFIX/bin/$HOST-$tool $PREFIX/bin/$OLD_HOST-$tool || true;
 done
-
-rm $PREFIX/bin/$HOST-ld || true;
-rm $PREFIX/bin/$OLD_HOST-ld || true;
-rm $PREFIX/$OLD_HOST/bin/ld || true;
-rm $PREFIX/$HOST/bin/ld || true;
-
-#ln -s $PREFIX/$HOST $PREFIX/$OLD_HOST
+© 2020 GitHub, Inc.
```
Comment (beckermr) [inline recipe/install_binutils.sh:14]:
```suggestion
```

File: recipe/install_ld.sh [hunk 4]
```diff
@@ -6,12 +6,7 @@ cd build
...
-OLD_CHOST="${ctng_cpu_arch}-${ctng_vendor}-linux-gnu"
+CHOST="${ctng_cpu_arch}-${ctng_vendor}-linux-gnu"
 mkdir -p $PREFIX/bin
-mkdir -p $PREFIX/$OLD_CHOST/bin
-mkdir -p $PREFIX/$CHOST/bin
 cp $PWD/install/$PREFIX/bin/$CHOST-ld $PREFIX/bin/$CHOST-ld
-ln -s $PREFIX/bin/$CHOST-ld $PREFIX/bin/$OLD_CHOST-ld
-ln -s $PREFIX/bin/$CHOST-ld $PREFIX/$OLD_CHOST/bin/ld
-ln -s $PREFIX/bin/$CHOST-ld $PREFIX/$CHOST/bin/ld
+© 2020 GitHub, Inc.
```
Comment (beckermr) [inline recipe/install_ld.sh:12]:
```suggestion
```

**Comment by beckermr** [comment]
These conflict messages are insane:

```

Package binutils_impl_linux-aarch64 conflicts for:
--
9285 | binutils_impl_linux-aarch64==2.34=h326052a_5
9286 | gcc_impl_linux-aarch64==7.3.0=hfeefbbc_5 -> binutils_impl_linux-aarch64[version='>=2.29.1']
9287 |  
9288 | Package ld_impl_linux-aarch64 conflicts for:
9289 | gcc_impl_linux-aarch64==7.3.0=hfeefbbc_5 -> binutils_impl_linux-aarch64[version='>=2.29.1'] -> ld_impl_linux-aarch64[version='2.33.1\|2.34',build='h326052a_4\|h326052a_5\|h326052a_3\|h326052a_2\|h326052a_8']
9290 | ld_impl_linux-aarch64==2.34=h326052a_5
```

**Comment by beckermr** [comment]
blocked by https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/58

**Comment by isuruf** [comment]
This shouldn't have been merged without the symlinks given that this is the latest build.

**Comment by beckermr** [comment]
Yup. I’ll push a fix today.

**Comment by beckermr** [comment]
I put this build up to prevent binutils downgrades with the latest compilers that don’t have the sysroot package.

**Pull request merged**